packages/gatsby-remark-copy-linked-files/src/index.js

```diff
@@ -58,10 +58,7 @@ const newPath = (linkNode, options) => {
 const newLinkURL = (linkNode, options, pathPrefix) => {
   const { destinationDir } = options
   const destination = getDestination(linkNode, destinationDir)
-  const linkPaths = [`/`, pathPrefix, destination].filter(lpath =>
-    lpath ? true : false
-  )
-  return path.posix.join(...linkPaths)
+  return `${pathPrefix ? pathPrefix : ``}/${destination}`
```

Comment:
Actual fix

Reply:
Is it possible that `pathPrefix` ends with `/`? If so, we may want to keep `path.posix.join`?

Reply:
When we read config this massages values to strip trailing slash:

https://github.com/gatsbyjs/gatsby/blob/8ad565f61ddd64f965513ef1d791c595629740d1/packages/gatsby/src/joi-schemas/joi.ts#L17-L34

This is done to avoid having to deal with quirks like that in all the plugins

Reply:
Makes sense, thanks!